doc/install.text

```diff
@@ -0,0 +1,197 @@
+  by a Bias-subtracted master Dark. We will then create a Bias- and Dark-subtracted master Flat, which we then use 
+  to identify and trace the apertures of the fiber traces. The fiber traces from the Arc image are then extracted
+  and wavelength calibrated.
+
+  The data we want to reduce were observed/simulated on 2015-12-22 on spectrograph 2, arm r (red) at site S (Summit). 
+
+  The parameter "--rerun USERNAME/rmp" (substitute USERNAME with your name) specifies where to store
+  temporary pipeline outputs.
+  Note the parameter "--cores 1" at the end. This parameter is required by tasks which are parallelized.
+  Sometimes running the code in parallel can lead to problems (in most cases caused by the 3rd-party 
+  libraries used), so setting cores to 1 is a safe choice.
+
+constructBias.py $PFS_DATA --rerun USERNAME/tmp --id field=BIAS dateObs=2015-12-22 arm=r spectrograph=2 --calibId calibVersion=bias calibDate=2015-12-22 arm=r spectrograph=2 --cores 1
+
+
+* Now that we have a master bias we need to ingest that into our calibration database stored in
+  $PFS_DATA/CALIB/calibRegistry.sqlite3. The parameter "--validity 180" specifies that the calibration
+  images are valid for 180 days. We will need to repeat this step every time we create a new calibration
+  image for the butler to find it when its use is required by a later called task.
+
+genCalibRegistry.py --root $PFS_DATA/CALIB --camera PFS --validity 180
+
+
+* Now we can create a trimmed and scaled, Bias-subtracted master Dark and ingest that into our calibration registry.
+  Again, you need to substitute USERNAME with your name.
+  If you don't want Dark subtraction you can skip this step, but remember further down to turn Dark subtraction
+  off.
+
+constructDark.py $PFS_DATA --rerun USERNAME/tmp --id field=DARK dateObs=2015-12-22 arm=r spectrograph=2 --calibId calibVersion=dark calibDate=2015-12-22 arm=r spectrograph=2 --cores 1
+genCalibRegistry.py --root $PFS_DATA/CALIB --camera PFS --validity 180
+
+
+* Having the master Bias and Dark, we can now create our master Flat. Currently the master Flat is only used to 
+  trace the apertures of the fiber traces and to calculate the spatial profile for the optimal extraction.
+  Note that for the actual flat-fielding dithered Flats will be used in the near future, and this kind of Flat
+  here will be renamed to apDef (aperture definition).
+  In our data set only visit 5 is a flat, so specifying "--id visit=5" is all you need to specify for the butler
+  to find our flat. In case you  would want to reduce all Flats taken 2015-12-22 for spectrograph 2, red arm,
+  you would need to replace "visit=5" with "field=FLAT arm=r dateObs=2015-12-22 spectrograph=2".
+  If you want to turn off Dark subtraction you can either edit the config file $OBS_PFS_DIR/config/pfs/flat.py
+  or simply add "--config isr.doDark=False" to the reduceFlat command, what will override the parameter setting
```

Comment:
Sentence dropped as no Dark subtraction option removed from the quick-start guide